config/application.rb

```diff
@@ -111,7 +111,9 @@ def self.sphinx_local?
     end
 
     def self.versions(file = Rails.root.join('WAGON_VERSIONS'))
-      @versions ||= file.exist? ? file.read.lines.reject(&:blank?) : []
+      @versions ||= {}
```

Comment:
The only piece of code I found accessing `Rails.application.class.version` is in the sentry initializer. So the default argument `WAGON_VERSIONS` will not be read at any time, therefor could be removed

Reply:
I thought I would be used in the footer where we display the deployed wagon-versions and link the respective commits. If that is wrong, then it could be indeed removed.